pom.xml

```diff
@@ -544,6 +556,23 @@ limitations under the License.]]>
           </supportedProjectTypes>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.revapi</groupId>
+        <artifactId>revapi-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <goals>
+              <goal>check</goal>
+            </goals>
+            <configuration>
+              <analysisConfigurationFiles>
+                <!-- Present at the root of each module that doesn't skip this goal -->
+                <analysisConfigurationFile>revapi.json</analysisConfigurationFile>
```

Comment:
I like having a configuration file per module, if we need to add exclusions in the future, it's better to have them at the module level.